release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1070,6 +1101,17 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 
 * Previously, a race condition occurred between the handling of pods that had been scheduled and the pods that had been completed on a node when OVN-Kubernetes started. This condition often occurred when nodes rebooted. Consequently, the same IP was erroneously assigned to multiple pods. This update fixes the race condition so that the same IP is not assigned to multiple pods in those circumstances. (link:https://issues.redhat.com/browse/OCPBUGS-16634[*OCPBUGS-16634*])
 
+* Previously, there was an error that caused a route to be rejected due to a duplicate host claim. When this occurred, the system would mistakenly select the first route it encountered, which was not always the conflicting route. With this update, all routes for the conflicting host are first retrieved and then sorted based on their submission time. This allows the system to accurately determine and select the newest conflicting route. (link:https://issues.redhat.com/browse/OCPBUGS-16707[*OCPBUGS-16707*])
+
+* Previously, when a new `ipspec-host` pod was started, it would clear or remove the existing XFRM state. Consequently, it would remove existing north-south traffic policies. This issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-19817[*OCPBUGS-19817*])
```

Comment:
```suggestion
* Previously, when a new `ipspec-host` pod was started, it would clear or remove the existing `XFRM` state. Consequently, it would remove existing north-south traffic policies. This issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-19817[*OCPBUGS-19817*])
```

Reply:
https://man7.org/linux/man-pages/man8/ip-xfrm.8.html#:~:text=%5B%20OPTIONS%20%5D-,xfrm,-%7B%20COMMAND%20%7C 🤷